otel/otel-release-notes.adoc

```diff
@@ -29,6 +29,33 @@ Data enrichment and processing:: Before forwarding data to analysis tools, the C
 
 Multi-backend receiving and exporting:: The Collector can receive and send data to multiple monitoring and analysis platforms simultaneously.
 
+[id="otel-3-1-rn"]
+== {OTELName} 3.1
+
+{OTELName} 3.1 is based on link:https://opentelemetry.io/[OpenTelemetry] 0.93.0.
+
+[id="new-features-and-enhancements_otel-3-1-rn"]
+=== New features and enhancements
+
+This update introduces the following enhancements:
+
+* Enable target allocator in the OpenTelemetry collector.
```

Comment:
I'll suggest a short summary of what the target allocator is and what it does (https://opentelemetry.io/docs/kubernetes/operator/target-allocator/).